lsp-completion.el

```diff
@@ -715,6 +706,12 @@ The CLEANUP-FN will be called to cleanup."
   "Disable LSP completion support."
   (lsp-completion-mode -1))
 
+(defun lsp-completion-pass-through-try-completion (string table pred _point)
+  (try-completion string table pred))
+
+(defun lsp-completion-pass-through-all-completions (string table pred _point)
+  (all-completions string table pred))
```

Comment:
This is not correct here. It should be `(all-completions "" table pred)` in order to disable the prefix filtering.

Reply:
Furthermore - using `all-completions` does not apply highlighting. You have to add that separately based on the argument `string`.